source/Engine.cpp

```diff
@@ -590,6 +625,7 @@ void Engine::Step(bool isActive)
 		highlightSprite = flagship->GetSprite();
 		highlightUnit = flagship->Unit() * zoom;
 		highlightFrame = flagship->GetFrame();
+		highlightLocation = flagship->Position();
```

Comment:
```suggestion
		highlightLocation = flagship->Center();
```
? although I am not sure